Sources/GRPC/HTTP1ToGRPCServerCodec.swift

```diff
@@ -383,7 +379,14 @@ extension HTTP1ToGRPCServerCodec: ChannelOutboundHandler {
       }
 
       if contentType == .webTextProtobuf {
-        precondition(responseTextBuffer != nil)
+        var responseTextBuffer = self.responseTextBuffers.popFirst() ?? context.channel.allocator.buffer(capacity: 5)
+
+        // Append the rest of the buffers.
+        let requiredExtraCapacity = self.responseTextBuffers.map { $0.readableBytes }.reduce(0, +)
```

Comment:
May be worth using `.lazy` here to avoid the intermediate array of ints.

Reply:
Good suggestion, thanks.